src/lib/ci.js

```diff
@@ -1,7 +1,10 @@
 const globToRegExp = require('glob-to-regexp');
 const { getChangedFiles } = require("./utils");
 
-
+//isCIRequired checks if we need to run the Jenkins job(based to trigger only on comment) and comment the required phrase, which is defined in comments.yml
```

Comment:
please use [JSDoc](https://jsdoc.app) instead.
something like
```
/**
 * checks if we need to run the Jenkins job(based to trigger only on comment) and comment the required phrase, which is defined in comments.yml
 * @param {string} context - The context from which the PR is coming from
 * @returns {boolean} true/false depending on the method logic
 */
```

Reply:
the same for the rest of the methods (just the ones from `src/lib` folder)

Reply:
Okay changing.